serial_posix broken

Using go 1.17:

```
❯ go get -u
go: downloading github.com/grid-x/serial v0.0.0-20211028062334-07e419efa8cd
go: downloading golang.org/x/net v0.0.0-20211029224645-99673261e6eb
go: downloading github.com/russross/blackfriday v1.6.0
go: downloading golang.org/x/sys v0.0.0-20211029165221-6e7872819dc8
go: downloading github.com/kr/pretty v0.2.0
go: downloading gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15
# github.com/grid-x/serial
../../go/pkg/mod/github.com/grid-x/serial@v0.0.0-20211028062334-07e419efa8cd/serial_posix.go:116:9: undefined: io
```

https://github.com/grid-x/serial/blob/master/serial_posix.go is missing the `io` import. Should use `goimports` when editing ;)

--- a/serial_posix.go
+++ b/serial_posix.go
@@ -5,6 +5,7 @@ package serial
 import (
 	"errors"
 	"fmt"
+	"io"
 	"log"
 	"os"
 	"syscall"

